app/lib/bk/compat/parsers/circleci/executors.rb

```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
-require_relative '../../pipeline/plugin'
-require_relative '../../pipeline/step'
-
```

Comment:
This file still uses the circleci-specific command step so it should include the dependencies explicitly (remember to adapt the path if the file is moved/renamed as suggested):

```suggestion
require_relative '../../models/circleci/circleci_step'

```

Reply:
Adjusted - latter becomes `../../models/steps/circleci` 👍